src/fmu/dataio/_objectdata_provider.py

```diff
@@ -157,7 +159,7 @@ def _derive_name_stratigraphy(self) -> dict:
 
         """
         logger.info("Evaluate data:name attribute and stratigraphy")
-        result = {}  # shorter form
+        result: Dict[str, Any] = {}
```

Comment:
```suggestion
        result: dict[str, Any] = {}
```
Just for consistency, I think this is ok?